src/features/projects/CreateProjectForm.jsx

```diff
@@ -0,0 +1,156 @@
+import { useDispatch, useSelector } from 'react-redux';
+import { Formik, Form, Field } from 'formik';
+import * as Yup from 'yup';
+import { timeZonesNames } from '@vvo/tzdb';
+import _ from 'lodash';
+
+import { styled } from '../../theme/stitches.config.js';
+import { FormWrapper, FormFieldWrapper, FieldRow, ButtonRow, FormError } from '../../components/Form';
+import Button from '../../components/Button.jsx';
+import SelectField from '../../components/SelectField.jsx';
+import {
+  Toast,
+  ToastTitle,
+  ToastAction,
+  ToastViewport
+} from '../../components/Toast';
+import IconButton from '../../components/IconButton';
+import { Cross2Icon } from '@radix-ui/react-icons';
+import { createProject, selectCreateProjectState, dismissStateMsg } from './projectsSlice.js';
+
+const PageWrapper = styled('div', {
+  maxWidth: '600px',
+  padding: '0 $5',
+  width: '100%',
+  margin: '0 auto'
+});
+
+const Header = styled('div', {
+  fontSize: '24px',
+  fontWeight: '$5',
+  fontFamily: '$roboto',
+  paddingTop: '$8',
+  marginBottom: '$4'
+});
+
+const createProjectSchema = Yup.object().shape({
+  name: Yup.string().required('Enter a project name'),
+  description: Yup.string().required('Enter a short description'),
+  timezone: Yup.string().required('Select a timezone'),
+  availableMLModels: Yup.array().required('Select a ML model'),
+});
+
+const CreateProjectForm = () => {
+  const dispatch = useDispatch();
+  const stateMsg = useSelector(selectCreateProjectState);
+
+  const tzOptions = timeZonesNames.map((tz) => ({ value: tz, label: tz }));
+  const mlModelOptions = [
+    {value: 'megadetector_v5a', label: 'megadetector_v5a'},
+    {value: 'megadetector_v5b', label: 'megadetector_v5b'},
+    {value: 'mirav2', label: 'mirav2'}
+  ];
+
+  return (
+    <PageWrapper>
+      <Header>Create project</Header>
+      <FormWrapper>
```

Comment:
A loading spinner overlay would be nice here, e.g.: https://github.com/tnc-ca-geo/animl-frontend/blob/97b29d372733cf0bc2024561652896d3f86fcb49/src/features/projects/DeleteViewForm.jsx#L43